_posts/2017-05-16-official-docker-image.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title:  "Introducing official Docker images for Apache Flink"
+date:   2017-05-16 09:00:00
+author: "Patrick Lucas (Data Artisans) and Ismaël Mejía (Talend)"
+author-twitter: "iemejia"
+categories: news
+---
+
+For some time, the Apache Flink community has provided scripts to build a Docker image to run Flink. Now, starting with version 1.2.1, Flink will have an [official Docker image](https://hub.docker.com/r/_/flink/). This image is maintained by the Flink community and curated by the [Docker](https://github.com/docker-library/official-images) team to ensure it meets the quality standards for container images of the Docker community.
+
+A community-maintained way to run Apache Flink on Docker and other container runtimes and orchestrators is part of the ongoing effort by the Flink community to make Flink a first-class citizen of the container world.
+
+{:refdef: style="text-align: center;"}
+![Flink in a container](https://activerain-store.s3.amazonaws.com/image_store/uploads/8/7/6/3/9/ar12988558393678.JPG){:width="300px"}
```

Comment:
Sadly I don't think we can assume we have the rights to this image. (Source: http://activerain.com/blogsview/2160841/-heard-of-jack-in-the-box----how-about-squirrel-in-the-box----)

I haven't been able to find an equally-apropos picture with usage rights so I think we just have to nix this.